bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -24,165 +24,324 @@
 Workflow generator to run pygrb offline post-processing.
 """
 
-from __future__ import division
-
+import sys
+import socket
 import logging
 import argparse
 import os
 import pycbc.version
+from pycbc import init_logging
 import pycbc.workflow as _workflow
-from pycbc.workflow.plotting import PlotExecutable, excludestr, requirestr
-from pycbc.results import create_versioning_page, layout, pygrb_plotting_utils
+from pycbc.workflow.plotting import PlotExecutable
+from pycbc.results import create_versioning_page, layout, pygrb_postprocessing_utils
 from pycbc.results.versioning import save_fig_with_metadata
+from pycbc.workflow import pegasus_workflow as wdax
+from Pegasus import DAX3 as dax
+from six.moves.urllib.request import pathname2url
+from six.moves.urllib.parse import urljoin
```

Comment:
Please don't use `six` any more. Please just use the python3 versions.

Reply:
I've commented on this in a summary message below (March 4, 2022)